.github/workflows/conventional-pr.yml

```diff
@@ -18,3 +18,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           commitTitleMatch: "false"
+          ignoreCommits: true
```

Comment:
I changed the configuration of the squash & merge commit to match the title of the PR, so this is no longer necessary:

<img width="901" alt="image" src="https://github.com/user-attachments/assets/75b713ed-5855-4697-bbc1-ac2a27a6b922" />